.gitignore

```diff
@@ -163,3 +163,5 @@ $RECYCLE.BIN/
 .sass-cache/
 _site/
 .Rproj.user
+/examples/Intro to R
```

Comment:
Not sure why these lines are in the .gitignore. Can you remove these and resubmit the PR? thx.